grants/czi-eoss5-notebook-v7-loi.md

```diff
@@ -0,0 +1,75 @@
+## Proposal Title: Auto-filled; Maximum of 60 characters, including spaces.
+
+Jupyter Notebook maintenance and community support for v4 migration.
+
+## Amount Requested: Total budget amount requested in USD, including indirect costs; this number should be between $100,000 USD and $400,000 USD total costs over a two-year period.
+
+$400,000 USD
+
+## Proposal Summary/Scope of Work: Provide a short summary of the work being proposed (maximum of 500 words).
+
+JupyterLab and the Classic Notebook are the two major user interfaces to manipulate the Jupyter notebook documents. That format is really popular within the scientific community thanks to its ability to bring code, data analysis and visualization in a single document.
```

Comment:
```suggestion
JupyterLab and the Classic Notebook are the two major user interfaces to manipulate Jupyter notebook documents. This document format is popular within the scientific community because it brings code, data analysis, and visualization together in one document.
```